app/assets/stylesheets/application.scss

```diff
@@ -2,8 +2,25 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+$accent-color: #FF384D;
+$bg-color: #F9F9F9;
+$dark-bg: #464A57;
+$active-dark: #44474B;
+$active-accent: #FF2849;
+
 body {
-  background: #F9F9F9
+  background: $bg-color
+}
+
+body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
+  font-family: serif;
```

Comment:
Properties should be ordered color, font-family, font-weight, line-height<br>Each selector in a comma sequence should be on its own line